From: Andrew Morton <akpm@linux-foundation.org>
Subject: ia64-fix-clock_getresclock_monotonic-to-report-itc-frequency-checkpatch-fixes
Date: Sat Aug 20 02:01:10 PM PDT 2022

ERROR: switch and case should be at the same indent
#70: FILE: arch/ia64/kernel/sys_ia64.c:185:
+	switch (which_clock) {
+		case CLOCK_REALTIME:
+		case CLOCK_MONOTONIC:

WARNING: Missing a blank line after declarations
#75: FILE: arch/ia64/kernel/sys_ia64.c:190:
+			struct timespec64 rtn_tp = ns_to_timespec64(tick_ns);
+			return put_timespec64(&rtn_tp, tp);

total: 1 errors, 1 warnings, 37 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/ia64-fix-clock_getresclock_monotonic-to-report-itc-frequency.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Sergei Trofimovich <slyich@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 arch/ia64/kernel/sys_ia64.c |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

--- a/arch/ia64/kernel/sys_ia64.c~ia64-fix-clock_getresclock_monotonic-to-report-itc-frequency-checkpatch-fixes
+++ a/arch/ia64/kernel/sys_ia64.c
@@ -183,11 +183,11 @@ ia64_clock_getres(const clockid_t which_
 	 * clocks.
 	 */
 	switch (which_clock) {
-		case CLOCK_REALTIME:
-		case CLOCK_MONOTONIC:
-			s64 tick_ns = DIV_ROUND_UP(NSEC_PER_SEC, local_cpu_data->itc_freq);
-			struct timespec64 rtn_tp = ns_to_timespec64(tick_ns);
-			return put_timespec64(&rtn_tp, tp);
+	case CLOCK_REALTIME:
+	case CLOCK_MONOTONIC:
+		s64 tick_ns = DIV_ROUND_UP(NSEC_PER_SEC, local_cpu_data->itc_freq);
+		struct timespec64 rtn_tp = ns_to_timespec64(tick_ns);
+		return put_timespec64(&rtn_tp, tp);
 	}
 
 	return sys_clock_getres(which_clock, tp);
_
